xbmc/platform/linux/peripherals/PeripheralBusUSBLibUdev.cpp

```diff
@@ -81,6 +81,13 @@ CPeripheralBusUSB::CPeripheralBusUSB(CPeripherals& manager) :
 
   /* set up a devices monitor that listen for any device change */
   m_udevMon = udev_monitor_new_from_netlink(m_udev, "udev");
+
+  /* filter to only receie usb events */
+  if (udev_monitor_filter_add_match_subsystem_devtype(m_udevMon, "usb", NULL) < 0)
```

Comment:
I think the filter should correspond to the entries in peripherals.xml. Currently it is only usb but someone might add other devices there.

Reply:
Jep, let me check.